src/components/MapAnchor.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { Pressable, Platform } from "react-native";
+import * as Linking from "expo-linking";
+import { getMapAnchorA11yLabel, MAP_ANCHOR_A11Y } from "../util/strings";
+
+enum TravelMode {
+	Driving = "driving",
+	Walking = "walking",
+	Transit = "transit",
+}
+
+const handlePress = async (href: string) => {
+	await Linking.openURL(href);
+};
+
+const buildMapsHref = (
+	travelmode: TravelMode,
+	destination_place_id: string,
+	destination?: string,
+	formatted_address?: string
+) => {
+	return Platform.select<string>({
+		android: buildGoogleMapsHref(travelmode, false, destination, destination_place_id),
+		ios: buildAppleMapsHref(travelmode, destination, formatted_address),
+		default: buildGoogleMapsHref(travelmode, false, destination, destination_place_id),
+	});
+};
+
+const buildGoogleMapsHref = (
+	travelmode: TravelMode,
+	startNavigation: boolean,
+	destination?: string,
+	destination_place_id?: string
+) => {
+	// https://developers.google.com/maps/documentation/urls/get-started#directions-action
+	let href = "https://www.google.com/maps/dir/?api=1";
+	href += `&travelmode=${travelmode}`;
```

Comment:
sort of a nitpick but prob should either add or use the template literal here, not both
`let href = https://www.google.com/maps/dir/?api=1&travelmode=${travelmode}`

Reply:
or 
	let href = "https://www.google.com/maps/dir/?api=1&travelmode=";
	href += travelmode;

or
	let href = "https://www.google.com/maps/dir/?api=1";
	href += `&travelmode=`;
	href += travelmode;
your call but that was my gut instinct, just seemed a little weird to add something on the next line with a template literal in it